graspologic/layouts/auto.py

```diff
@@ -8,14 +8,15 @@
 
 import networkx as nx
 import numpy as np
-import umap
+import umap.umap_ as umap
```

Comment:
any reason for this change?

Reply:
i was originally getting a 'module umap has no attribute UMAP' error so this was the temporary work around. It was just an environment issue though, and I tried just 'import umap' after uninstalling/installing umap and umap-learn and it works fine now. Sorry about that, changing it back now.

Reply:
no worries, thanks